api/app.js

```diff
@@ -18,17 +18,27 @@ const swaggerUIOptions = {
 };
 
 //###[  Routers ]###
-const indexRouter = require('./index/indexRouter');
-const providerRouter = require('./provider/providerRouter');
-const programRouter = require('./program/programRouter');
-const statusRouter = require('./statuses/statusesRouter');
```

Comment:
I don't see a statusRouter added down below, is this something that still needs to be made or just gone completely?